### PR TITLE
chore: migrate from TSLint to ESLint

### PR DESCRIPTION
**Pull request by sosukesuzuki**

### CODE REVIEW

**Review by fisker — APPROVED** [review]
Looks good, but let's merge after #233, I don't want to deal with conflicts now.

**Comment by fisker** [comment]
@sosukesuzuki Can you rebase?

**Comment by fisker** [comment]
Can you find a rule to enforce `.js` extension in the import source? Maybe [`import/extensions`](https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/extensions.md)? Recommended here https://github.com/typescript-eslint/typescript-eslint/issues/3288#issuecomment-819613449

File: package.json [hunk 1]
```diff
@@ -15,9 +15,13 @@
   "license": "MIT",
   "scripts": {
     "prepublish": "yarn run build",
-    "fix": "yarn prettier . --write",
-    "lint": "yarn prettier . --check && tslint -p ./tsconfig.json",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "lint": "run-p lint:*",
+    "lint:eslint": "eslint --ext=.ts .",
```
Comment (fisker) [inline package.json:20]:
```suggestion
    "lint:eslint": "eslint --ext=.ts,.js,.cjs .",
```

Currently we have `.js` and `.cjs` too, let's add them all.

**Comment by fisker** [comment]
Forgot delete `tslint.json`..

**Comment by fisker** [comment]
https://github.com/prettier/angular-estree-parser/commit/ac870f0e82cd64737a954d3c582e9667f222e9ea

**Comment by sosukesuzuki** [comment]
Ah thanks

**Pull request merged**